### ISSUE TITLE
Cannot start the local conference again after ending for the first time

### ISSUE THREAD

**Issue by powerqian** [BelledonneCommunications/liblinphone#174]
Hi,

I found an issue with the linphone-sdk that for the local conference, it is okay to start the local conference with multiple calls for the first time. However after ending the local conference, it cannot be started again, and prompting something like
```
liblinphone/Unable to add participant to conference 0x7fb826e37480 because it is in state TerminationPending
```
Seems like after the conference ended and changed the state to Terminated or TerminationPending, it never reset to the correct state, and for the all future attempts to start another local conference, it will fail by checking the state.

**Comment by attermann** [comment]
FYI, I'm having the same issue and have included the results of my debugging in #186.

**Comment by powerqian** [comment]
Addressed in https://github.com/BelledonneCommunications/linphone-sdk/issues/143#event-6154854981.
